assets/src/scss/layout/_navbar.scss

```diff
@@ -143,6 +179,7 @@ $navbar-default-height: 60px;
   }
 
   @include large-and-up {
+    display: none;
```

Comment:
As the frontend test notices https://app.circleci.com/pipelines/github/greenpeace/planet4-master-theme/4756/workflows/4ba298db-80d2-40d7-88b5-4c74300a797a/jobs/28292 there are 2 display declarations here

Reply:
Yes! I figured out today about this. I just updated the PR.